README.md

```diff
@@ -1,7 +1,8 @@
 # Microsoft's C++ Standard Library
 
 This is the official repository for Microsoft's implementation of the C++ Standard Library (also known as the STL),
-which ships as part of the MSVC toolset and the Visual Studio IDE.
+which ships as part of the MSVC toolset and the Visual Studio IDE. Our [Changelog][] tracks which updates to this

```

Comment:
Maybe we should say "meaningful" updates since we don't put everything there.

Also maybe we should use GitHub releases instead?

Reply:
I agree with Billy's suggestion to use releases eventually. It's a nice and clean way for users to see what the state of the code was when we shipped different VS releases (though of course it can't handle us backporting unless we start cutting branches on GitHub to represent our different supported releases).